pages/home.md

```diff
@@ -56,18 +56,18 @@ graphic_list:
 
 <section class="usa-section bg-base-lightest showcase" aria-label="Sites using USWDS">
```

Comment:
## Optional
Since we are updating the h2 to be more descriptive, it is _probably_ safe to remove the aria label on this section. In VoiceOver, I now hear "Sites using USWDS, region" immediately followed by "heading level 2, sites built with USWDS". No action is needed here unless @amycole501 or @alex-hull weigh in to agree. 

```suggestion
<section class="usa-section bg-base-lightest showcase">
```